stdlib/http/src/main/ballerina/src/http/non_listening_service_endpoint.bal

```diff
@@ -44,13 +44,9 @@ public type MockListener object {
         return self.detach(s);
     }
 
-    public function __init(int port, ListenerConfiguration? config = ()) {
+    public function init(int port, ListenerConfiguration? config = ()) {
         self.config = config ?: {};
         self.port = port;
-        self.init(self.config);
-    }
-
-    public function init(ListenerConfiguration c) {
```

Comment:
Looks like we are losing a public function here.

Reply:
I asked from stdlib-team and merged init and __init functions.